example/src/screens/ViewScreen.tsx

```diff
@@ -52,13 +52,7 @@ export function ViewScreenImpl() {
   }, [isUpdating])
 
   return (
-    <View style={[styles.container, { paddingTop: safeArea.top }]}>
-      <Text style={styles.header}>View</Text>
-      <View style={styles.topControls}>
-        <View style={styles.flex} />
-        <Text style={styles.buildTypeText}>{NitroModules.buildType}</Text>
-      </View>
-
+    <View style={[styles.container]}>
```

Comment:
Why did you remove this from here (the actual implementation of the screen) down to the wrapper? What is the benefit?